core/app/models/spree/promotion/rules/item_quantity.rb

```diff
@@ -0,0 +1,51 @@
+# A rule to apply to an order greater than (or greater than or equal to)
+# a specific quantity of line items
+module Spree
+  class Promotion
+    module Rules
+      class ItemQuantity < PromotionRule
+        preference :quantity_min, :integer, default: 1
+        preference :operator_min, :string, default: '>'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.